data/languages.yaml

```diff
@@ -1811,3 +1811,124 @@ ph:
         - kahapon: 1 araw
         - ngayon: 0 segundo
         - isang araw: 2 araw
+
+
+be:
+    name: Belarussian
+
+    skip: ["каля", "у", "ў", "і"]
+
+    monday:
+        - Панядзелак
+        - Пнд
+        - Пн
+    tuesday:
+        - Аўторак
+        - Аўт
+        - Аў
+    wednesday:
+        - Серада
+        - Сер
+        - Ср
+    thursday:
+        - Чацвер
+        - Чцв
+        - Чв
+    friday:
+        - Пятніца
+        - Пят
+        - Пт
+    saturday:
+        - Субота
+        - Суб
+        - Сб
+    sunday:
+        - Нядзеля
+        - Няд
+        - Нд
+
+    january:
+        - Студзеня
+    february:
+        - Лютага
+    march:
+        - Сакавіка
+    april:
+        - Красавіка
+    may:
+        - Мая
+        - Траўня
+    june:
+        - Чэрвеня
+    july:
+        - Ліпеня
+    august:
+        - Жнівеня
+        - Жніўня
+    september:
+        - Верасня
+    october:
+        - Кастрычніка
+    november:
+        - Лістапада
+    december:
+        - Снежня
+
+    year:
+        - год
+        - гады
+        - года
+        - гадоў
+        - г.
+    month:
+        - месяц
+        - месяца
+        - месяцы
+        - месяцаў
+    week:
+        - тыдзень
+        - тыдня
+        - тыдні
+        - тыдняў
+    day:
+        - дзень
+        - дні
+        - дзён
+        - дзен
+    hour:
+        - гадзіна
+        - гадзіны
+        - гадзіну
+        - гадзін
+    minute:
+        - хвіліна
+        - хвілін
+        - хвіліны
+        - хвіліну
+        - хвіл.
+        - хв.
+    second:
+        - секунда
+        - секунды
+        - секунду
+        - секунд
+
+    ago:
+        - таму назад
+        - таму
+        - назад
+
+    simplifications:
+        - пазаўчора: 2 дні
+        - учора: 1 дзень
+        - ўчора: 1 дзень
+        - сення: 0 дзён
+        - сёння: 0 дзён
+        - хв: хвіліну
+        - сек: секунду
+        - ^гадзіна: 1 гадзіна
+        - ^гадзіну: 1 гадзіну
+        - ^хвіліну: 1 хвіліну
+        - ^секунду: 1 секунду
+        - некалькі секунд: 44 секунды
```

Comment:
I'm not sure about leaving in grammatically incorrect forms. It will increase recall, but I can't remember any precedents in other languages, e.g. there's no "сикунд" in Russian. @Allactaga what do you think?